### DISCUSSION TITLE
[Resolved] Do we ask for questions here?

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#1] [Q&A] [answered]
Are we supposed to ask for questions related to PA1 (similar to piazza) here in the discussion section? or is there another platform where we are supposed to ask questions related to PA1?

**Comment by DelusionArun** [comment]
Just saw the new announcement in canvas

**Comment by Troublor** [comment, marked as answer]
**Of course!. Please ask questions in the [Discussions](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version/discussions) of this repo.** 

Questions and answers here can be viewed by all classmates. This can help eliminate a lot of duplicate questions asked by classmates. 

The [Issue](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version/issues) page is meant to track bugs in the skeleton code. 
Normally you should not ask questions there unless you find a bug in the skeleton code or obfuscated jar.

**Comment by Troublor** [comment]
Also other general questions, not necessarily related PAs, are also welcomed to asked in the Github [Discussions](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version/discussions).
